docsite/source/rules.html.md

```diff
@@ -181,6 +181,23 @@ contract.call(email: 'jane@doe.org', login: 'jane', password: "").errors.to_h
 # => {:password=>["password is required"]}
 ```
 
+The `key?` method supports passing an explicit key name for rules that have multiple keys.
+
+```ruby
+class DistanceContract < Dry::Validation::Contract
```

Comment:
Can we have a spec at contract level (/spec/integration/contract/) mimicking this same scenario where we are asking of an specific keys?

Reply:
Sure, I think the test coverage is pretty comprehensive already but that would probably better communicate the intent of this.

Reply:
I merged it as-is because having this example in the docs already covers the intended usage and the implementation is so simple that the unit test will suffice.